tests/fontInfoData_test.py

```diff
@@ -104,6 +105,16 @@ def test_caret_slope(self):
         self.assertEqual(
             getAttrWithFallback(info, "openTypeHheaCaretSlopeRun"), 200)
 
+    def test_head_created(self):
+        info = TestInfoObject()
+
+        os.environ["SOURCE_DATE_EPOCH"] = '1514485183'
+        self.assertEqual(
+            getAttrWithFallback(info, "openTypeHeadCreated"), '2017/12/28 18:19:43')
+        del os.environ["SOURCE_DATE_EPOCH"]
```

Comment:
Please use a try:... finally:... block here to ensure the environment is reset whether or not an exception occurs inside.

Reply:
I mean, like that:

```python
os.environ["SOURCE_DATE_EPOCH"] = '1514485183'
try:
    self.assertEqual(...)
finally:
    del os.environ["SOURCE_DATE_EPOCH"]
```